src/Microsoft.ML.Data/Transforms/ColumnConcatenatingEstimator.cs

```diff
@@ -10,9 +10,28 @@
 
 namespace Microsoft.ML.Transforms
 {
+
     /// <summary>
-    /// Concatenates columns in an <see cref="IDataView"/> into one single column. Estimator for the <see cref="ColumnConcatenatingTransformer"/>.
+    /// Concatenates one or more input columns into a new output column.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type | Any, except [key type](<xref:Microsoft.ML.Data.KeyDataViewType>). All input columns must have the same type.  |
+    /// | Output column data type | A vector of the input columns' data type |
+    ///
+    /// The resulting [ColumnConcatenatingTransformer](<xref:Microsoft.ML.Data.ColumnConcatenatingTransformer>) creates a new column,
```

Comment:
>[ColumnConcatenatingTransformer](<xref:Microsoft.ML.Data.ColumnConcatenatingTransformer>) [](start = 22, length = 89)

if the anchor text is the same as the class/type name, you can save some time by using autolinks:

[ColumnConcatenatingTransformer](xref:Microsoft.ML.Data.ColumnConcatenatingTransformer)   -----> <xref:Microsoft.ML.Data.ColumnConcatenatingTransformer>


also with anchor text the link should be [anchor](xref:uid) without any <>. The one you have won't work.